src/coreclr/jit/gentree.cpp

```diff
@@ -24365,6 +24365,54 @@ GenTree* Compiler::gtNewSimdMaxNode(
             return gtNewSimdHWIntrinsicNode(type, op1, op2, gtNewIconNode(0x05), minMaxIntrinsic, simdBaseJitType,
                                             simdSize);
         }
+        if (IsBaselineVector512IsaSupportedOpportunistically())
```

Comment:
While we're fixing `Max`, can you also fix `Min` with the similar optimization?

If it's trivial to do, then fixing the other Min/Max variants would also be beneficial, as they appear to be similarly accelerated for scalar but not taking advantage of AVX512 (vrange/vfixupimm) for vector